migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,82 @@
+
+[id="mtc-3-to-4-overview-differences-mtc"]
+== Differences between {product-title} 3 and 4
+Before migrating from {product-title} 3 to 4, you can check xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#planning-migration-3-4[differences between {product-title} 3 and 4]. Review the following information:
+
+* xref:../architecture/architecture.adoc#architecture[Architecture]
+* xref:../architecture/architecture-installation.adoc#architecture-installation[Installation and update]
+* xref:../storage/index.adoc#index[Storage], xref:../networking/understanding-networking.adoc#understanding-networking[network], xref:../logging/cluster-logging.adoc#cluster-logging[logging], xref:../security/index.adoc#index[security], and xref:../monitoring/monitoring-overview.adoc#monitoring-overview[monitoring considerations]
+
+[id="mtc-3-to-4-overview-planning-network-considerations-mtc"]
+== Planning network considerations
+Before migrating from {product-title} 3 to 4, review the xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#planning-migration-3-4[differences between {product-title} 3 and 4] for information about the following areas:
+
+* xref:../migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc#dns-considerations_planning-considerations-3-4[DNS considerations]
+** xref:../migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc#migration-isolating-dns-domain-of-target-cluster-from-clients_planning-considerations-3-4[Isolating the DNS domain of the target cluster from the clients].
+** xref:../migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc#migration-setting-up-target-cluster-to-accept-source-dns-domain_planning-considerations-3-4[Setting up the target cluster to accept the source DNS domain].
+
+You can migrate stateful application workloads from {product-title} 3 to 4 at the granularity of a namespace. To learn more about MTC see xref:../migrating_from_ocp_3_to_4/about-mtc-3-4.adoc#about-mtc-3-4[Understanding MTC].
+
+[NOTE]
+====
+If you are migrating from {product-title} 3, see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[About migrating from {product-title} 3 to 4] and xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-installing-legacy-operator_installing-3-4[Installing the legacy {mtc-full} Operator on {product-title} 3].
+====
+
+[id="mtc-overview-install-mtc"]
+== Installing MTC
+Review the following tasks to install the MTC:
+
+. xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-installing-mtc-on-ocp-4_installing-3-4[Install the {mtc-full} Operator on target cluster by using Operator Lifecycle Manager (OLM)].
+. xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-installing-legacy-operator_installing-3-4[Install the legacy {mtc-full} Operator on source cluster manually].
```

Comment:
on source cluster > on the source cluster

Reply:
Addressed